app/schemas/models/interactives/common/interactive_types.schema.js

```diff
@@ -15,27 +16,29 @@ const interactiveDraggableOrderingSchema = {
         additionalProperties: false,
         properties: {
           text: { type: 'string' },
-          elementId: schema.objectId({ hidden: true })
+          elementId: schema.stringID({ readOnly: true })
         }
       }
     },
-    solution: solutionSchema.elementOrderingSolutionSchema
+    solution: solutionSchema.elementOrderingSolutionSchema,
+    i18n: { type: 'object', format: 'i18n', props: ['labels', 'elements'], description: 'Help translate this interactive.' }
   }
 }
 
 const interactiveInsertCodeSchema = {
   type: 'object',
   additionalProperties: false,
+  title: 'Insert Code interactive data',
   properties: {
-    starterCode: { type: 'string' }, // codeLanguage will be determined by unitCodeLanguage in interactives schema
+    starterCode: { type: 'string', format: 'ace' }, // codeLanguage will be determined by unitCodeLanguage in interactives schema
```

Comment:
You probably know this. But I only recently figured out that the format is used to specify how the treema editor [displays the data](https://github.com/codecombat/codecombat/blob/351e96a0840d124262650d7ecac10eb4b5bb95a5/app/core/treema-ext.coffee#L548)!

Reply:
You are correct, I added this only for the editor frontend so that treema displays this like a code editor.